source/ways-of-working/new-component/feature-flags.html.md.erb

```diff
@@ -0,0 +1,35 @@
+---
+title: Feature flags
+last_reviewed_on: 2021-10-29
+review_in: 6 months
+weight: 99
+---
+
+# <%= current_page.data.title %>
+
+Feature flags are a software development and delivery technique that allows software teams to enable and disable parts of a codebase at the flip of a switch.
+Feature flags enable software teams to adopt DevOps practices, in so much as they help increase deployment speeds, system stability, and cross-team collaboration.
+Feature flags also allow you to decouple code deployments from feature releases. You can make code changes in production while hiding those changes from users.
```

Comment:
Maybe include that it's an enabler of trunk-based development? https://trunkbaseddevelopment.com/